md/defaults.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python3
+
+import pathlib
+
+print(
+"""\
+css: {datadir}/pandoc.css
+filters: [{datadir}/pandoc.py]
+
+metadata:
+  highlighting:
+    inline-code: cpp\
```

Comment:
This allows `code_cpp` to go away.

Reply:
Is this a new pandoc thing or something that always existed and I just didn't know about?

Reply:
No, I implemented it yesterday: https://github.com/mpark/wg21/commit/f23a31ba796b88248003d4de809c6412648862e0